value.go

```diff
@@ -82,6 +82,11 @@ func NewValue(iso *Isolate, val interface{}) (*Value, error) {
 		rtnVal = &Value{
 			ptr: C.NewValueNumber(iso.ptr, C.double(v)),
 		}
+	case []uint8: // TwinTag added
+		rtnVal = &Value{
+			//NOTE: C.CBytes() allocates memory, must be freed
```

Comment:
How will CBytes be freed

Reply:
The newly added function C.NewValueUint8Array() creates a BackingStore that takes ownership over the array allocated here by C.CBytes(). This allocation gets freed when V8 decides it can get rid of the BackingStore. Verified manually by adding a print statement to the deallocator.

Reply:
Would/Could this cause a panic on the Go side? For example V8 removes the backing store, and then the caller tries to read from the Go slice.

Reply:
The bytes allocated by C.CBytes() are passed directly into the V8 BackingStore that then takes ownership over the allocation. They are not exposed or handed out anywhere else, so it is not possible for any Go code to do an invalid access.

Reply:
This comment in the code is not very useful at the moment.

What is important from the perspective of this function is that `NewValueUint8Array` takes ownership of that memory allocation.

```suggestion
			// C.NewValueUint8Array takes ownership of the C.CBytes() allocation, making sure it gets freed.
```

Details of how C.NewValueUint8Array does that matter more in the context of its implementation.  I think it is clear enough from looking at that function that it is passing a deletion callback to the backing store to free the memory.  If you disagree, please comment on that code.